### ISSUE TITLE
newbie

### ISSUE THREAD

**Issue by machine4life** [machine4life/recommenderSystem#3]
1. python2 and python3
https://www.cnblogs.com/thunderLL/p/6643022.html
2. pip3 install xgboost报错，在这里下载了whl，然后D:\Program Files\Python\common>pip install xgboost-0.7-cp36-cp36m-win_amd64.whl，更新（还是没解决，无法import）
https://www.lfd.uci.edu/~gohlke/pythonlibs/#xgboost
 3. 解决了第二个问题。将Windows XP/WIN7系统，将dll复制到C:\Windows\System32目录下。
下载地址为：
http://www.jb51.net/dll/vcomp140.dll.html
4. python xgboost （备注python36）
https://xgboost.readthedocs.io/en/latest/get_started/
没有('demo/data/agaricus.txt.train') github克隆下来之后就有了
Administrator@SZ-20171205CSVD MINGW64 /d/code/xgboost (master)
$ git clone https://github.com/dmlc/xgboost --recursive^C
报错feature_names mismatch XGBoost，其实是测试集整体没有覆盖全126个特征，不是指某一条。
http://blog.csdn.net/lujiandong1/article/details/52743396
5. python sklearn logistic regression （备注python27）
http://scikit-learn.org/stable/auto_examples/linear_model/plot_logistic_path.html#sphx-glr-auto-examples-linear-model-plot-logistic-path-py
6. tensorflow无法访问
https://www.jianshu.com/p/3976adc80fc3
在hosts文件中添加    64.233.188.121 www.tensorflow.org 即可. ##且不要问64.233.188.121是什么
7. TensorFlow自带wdl
  pip3 intall tensorflow
python代码在这里还可以找到并成功运行 来自这篇博客http://www.cnblogs.com/zhanxiage1994/p/7489537.html
https://github.com/tensorflow/tensorflow/edit/r1.3/tensorflow/examples/learn/wide_n_deep_tutorial.py
python3 wide_n_deep_tutorial.py

**Comment by machine4life** [comment]
2.1
D:\Program Files\Python>cd common 
D:\Program Files\Python\common>pip install xgboost-0.7-cp36-cp36m-win_amd64.whl
Processing d:\program files\python\common\xgboost-0.7-cp36-cp36m-win_amd64.whl
Requirement already satisfied: scipy in d:\program files\python\python36\lib\sit
e-packages (from xgboost==0.7)
Requirement already satisfied: numpy in d:\program files\python\python36\lib\sit
e-packages (from xgboost==0.7)
Installing collected packages: xgboost
Successfully installed xgboost-0.7
2.2
D:\Program Files\Python\common>
![image](https://user-images.githubusercontent.com/33926586/35727187-40f6c5c4-0842-11e8-8772-fb520bb5ed13.png)

**Comment by machine4life** [comment]
脚本之家下载-dll下载频道，帮您搞定文件丢失问题

www.jb51.net/dll/部分dll文件中有多个目录分别表示多个系统专用文件

X86表示32位系统
x64表示64位系统

dll控件常规安装方法（仅供参考）：

一、如果在运行某软件或编译程序时提示缺少、找不到dll等类似提示，您可将从脚本之家下载来的dll拷贝到指定目录即可(一般是system系统目录或放到软件同级目录里面)，或者重新添加文件引用。 
二、直接拷贝该文件到系统目录里：
　　 1、Windows 95/98/Me系统，将dll复制到C:\Windows\System目录下。
　　 2、Windows NT/2000系统，将dll复制到C:\WINNT\System32目录下。
　　 3、Windows XP/WIN7系统，将dll复制到C:\Windows\System32目录下。
三、打开"开始-运行-输入regsvr32 dll"，回车即可解决。希望脚本之家为您提供的dll对您有所帮助。 

通过脚本之家下载dll的朋友，可将下面的代码保存为“注册.bat“，放到dll目录，就会自动完成dll注册(win98不支持)。



@echo 开始注册
copy dll %windir%\system32\
regsvr32 %windir%\system32\dll /s
@echo dll注册成功
@pause

**Comment by machine4life** [comment]
7. Microsoft Windows [版本 6.1.7601]
版权所有 (c) 2009 Microsoft Corporation。保留所有权利。

C:\Users\Administrator>d:

D:\>cd code

D:\code>cd web

D:\code\web>dir
 驱动器 D 中的卷没有标签。
 卷的序列号是 6C26-3BD2

 D:\code\web 的目录

2018/02/05  11:15    <DIR>          .
2018/02/05  11:15    <DIR>          ..
2018/02/03  09:10             1,195 plot_logistic_path.py
2018/02/05  11:15             8,225 wide_n_deep_tutorial.py
               2 个文件          9,420 字节
               2 个目录 276,871,376,896 可用字节

D:\code\web>python3 wide_n_deep_tutorial.py
Training data is downloaded to C:\Users\ADMINI~1\AppData\Local\Temp\tmp0ymc3xqo
Test data is downloaded to C:\Users\ADMINI~1\AppData\Local\Temp\tmpfivq1p8j
2018-02-05 11:16:08.997030: I C:\tf_jenkins\workspace\rel-win\M\windows\PY\36\te
nsorflow\core\platform\cpu_feature_guard.cc:137] Your CPU supports instructions
that this TensorFlow binary was not compiled to use: AVX AVX2
WARNING:tensorflow:enqueue_data was called with num_epochs and num_threads > 1.
num_epochs is applied per thread, so this will produce more epochs than you prob
ably intend. If you want to limit epochs, use one thread.
WARNING:tensorflow:enqueue_data was called with shuffle=False and num_threads >
1. This will create multiple threads, all reading the array/dataframe in order.
If you want examples read in order, use one thread; if you want multiple threads
, enable shuffling.
model directory = C:\Users\ADMINI~1\AppData\Local\Temp\tmpte27hivh
accuracy: 0.8353909
accuracy_baseline: 0.76377374
auc: 0.8745185
auc_precision_recall: 0.7114024
average_loss: 0.37545654
global_step: 2000
label/mean: 0.23622628
loss: 37.50189
prediction/mean: 0.26989672

D:\code\web>
